### PR TITLE
[bluetooth] Service crash container at startup

### LINKED ISSUE #898: [bluetooth] Service crash container at startup
I'm running amd64 container. ( no bluetooth on host )
```
> gladys-server@ start:prod /src/server
> cross-env NODE_ENV=production node index.js

Initialising OpenZWave 1.6.1051 binary addon for Node.JS.
        OpenZWave Security API is ENABLED
        ZWave device db    : /usr/local/etc/openzwave
        User settings path : /src/server/services/zwave/node_modules/openzwave-shared/build/Release/../../
        Option Overrides : --Logging false --ConsoleOutput false --SaveConfiguration true
2020-10-16T22:22:24+0200 <info> index.js:13 (Object.start) Starting usb service
2020-10-16T22:22:24+0200 <info> index.js:16 (Object.start) Starting zwave service
2020-10-16T22:22:24+0200 <info> index.js:15 (Object.start) Starting Bluetooth service
2020-10-16T22:22:24+0200 <info> index.js:19 (Object.start) Starting telegram service
2020-10-16T22:22:24+0200 <info> index.js:20 (Object.start) Starting Open Weather service

/src/server/services/bluetooth/node_modules/@abandonware/noble/lib/hci-socket/hci.js:100
    this._deviceId = this._socket.bindRaw(deviceId);
                                  ^

Error: ENODEV, No such device
    at Hci.init (/src/server/services/bluetooth/node_modules/@abandonware/noble/lib/hci-socket/hci.js:100:35)
    at NobleBindings.init (/src/server/services/bluetooth/node_modules/@abandonware/noble/lib/hci-socket/bindings.js:78:13)
    at /src/server/services/bluetooth/node_modules/@abandonware/noble/lib/noble.js:60:24
    at processTicksAndRejections (internal/process/task_queues.js:79:11) {
  errno: 19,
  code: 'ENODEV',
  syscall: 'bind'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gladys-server@ start:prod: `cross-env NODE_ENV=production node index.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gladys-server@ start:prod script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-10-16T20_07_23_616Z-debug.log

```

### PR DESCRIPTION
**Pull request by atrovato**
Fixes #898

### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [x] If your changes affects code, did your write the tests?
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- ~~[ ] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)~~
- ~~[ ] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)~~
- ~~[ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).~~
- ~~[ ] Did you add fake requests data for the demo mode (`front/src/config/demo.json`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).~~

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

Add global error handler.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -15,6 +15,11 @@ process.on('unhandledRejection', (error, promise) => {
   logger.error(error);
 });
 
+process.on('uncaughtException', (error, promise) => {
```
Comment (Pierre-Gilles) [inline server/index.js:18]: Why did you add that again? We already have it ! 

Screenshot 2020-11-02 at 10 22 39

File: server/migrations/20201031091221-service-reword-status.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.changeColumn('t_service', 'status', {
```
Comment (Pierre-Gilles) [inline server/migrations/20201031091221-service-reword-status.js]: This will drop the table, same problem as removeColumn

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
Edited: my bad, I understand the fixes now I read too fast.

Be careful with the changeColumn again, it destroys the table!

File: server/migrations/20201031091221-service-reword-status.js (hunk 2 above)
Comment (atrovato) [reply server/migrations/20201031091221-service-reword-status.js]: :'( we have to live with the type for ever?

Comment (Pierre-Gilles) [reply server/migrations/20201031091221-service-reword-status.js]: We have at least to wait that Sequelize improve the way they do a change/remove/rename column! I'm looking forward to their v6 which is in beta. 

But for now, those methods are not authorized in Gladys. I need to add another check for this function too

Comment (atrovato) [reply server/migrations/20201031091221-service-reword-status.js]: I add changeColumn restriction.

Comment (atrovato) [reply server/migrations/20201031091221-service-reword-status.js]: And keep workaround.

Comment (Pierre-Gilles) [reply server/migrations/20201031091221-service-reword-status.js]: Thanks for the changes!

**Pull request merged**